dockertest/dockercmd.py

```diff
@@ -379,6 +422,27 @@ def process_id(self):
                                   " called.")
         return self._async_job.sp.pid
 
+    @property
+    def container_id(self):
+        """
+        Try to discover our own container ID or name. Return None if we can't.
+        """
```

Comment:
Optional idea:  Maybe add a 'cidfile=None' parameter.  When non-None, if the file exists, read it and return.  This will be much faster than going searching for the CID through the process tree (assuming there are a huge number of processes/containers to search).

Another idea: Optional parameter for the container's name.  Looking up the CID by search of a ``dockertest.DockerContainers.list_containers()`` list is much faster than a potential boat-load of ``docker inspect`` calls.